index.js

```diff
@@ -61,10 +73,13 @@ const resolver = Consumer.create({
         const caseID = data.createcaseresponse.caseid;
 
         logger.info({ caseID, message: 'Casework submission successful' });
-
+        submitDuplicate({duplicate: false, case_id: caseID, external_id: externalId})
+          .catch(err => logger.log('error', err.message));
         return submitAudit({ success: true, caseID });
       }
       logger.info({ externalId, message: `Case already submitted with iCasework Case ID ${icwID}` });
+      submitDuplicate( {duplicate: true, case_id: icwID, external_id: externalId })
+        .catch(err => logger.log('error', err.message));
```

Comment:
@Alex-Swann this is to prevent the following issue occurring

{"level":"error","message":"Duplicate submission failed: 5xx - error: insert into \"duplicates\" (\"case_id\", \"duplicate\", \"external_id\") values ($1, $2, $3) - relation \"duplicates\" does not exist"}
(node:45535) UnhandledPromiseRejectionWarning: Error: Duplicate Table Error
    at submitDuplicate (/Users/christophermedland/HO/SAS/HOF/icasework-resolver/index.js:56:11)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)